chart/templates/dashboard-deployment.yaml

```diff
@@ -50,8 +50,8 @@ spec:
         readinessProbe:
           failureThreshold: 3
           httpGet:
-            path: /schemas/gitpod-schema.json
-            port: 80
+            path: /ready
```

Comment:
that would only mean "the http service is running", the previous check means "the dashboard is served". in-between there might be error cases. 

@geropl, do you know why we started to use actual resources from the dashboard to be tested for in the readiness probe?

Reply:
I can imagine several scenarios why that was:
- convenience: by testing well known dashboard resources, we don't have to think about other paths that we might want to use
- previous breakage: we may have had some misconfiguration at some point and then moved to checking a dashboard resource

Either way, once caddy is running it would serve the dashboard. It makes sense to me to use a readiness endpoint of caddy here.